### DISCUSSION TITLE
Is it possible to use this to create a dashboard to show insight?

### DISCUSSION THREAD

**Discussion by kmthorsnes** [flowwer-dev/pull-request-stats#82] [Q&A]
HI, 
I understand that it can be possible to use this to create insight/metrix for a team across repos. 
But it it possible to generate the stats other places than in a PR ? 
Can the report be generated to be somewhere else? 

What I would love to achieve would be to create a dashboard where i can see stats for the teams PR's across repos.

**Comment by manuelmhtr** [comment]
Hi @kmthorsnes 
There are some integrations: Slack, Teams and webhooks.

I'm also working on a web version but will take a while to be ready ;)

**Reply by kmthorsnes** [reply]
Thank you for replying :) Looking forward to following this!
